File: kotlin/raindrops/src/main/kotlin/Raindrops.kt
object Raindrops {

    fun convert(n: Int): String {
        var outputString = ""
        if (n % 3 === 0) outputString += "Pling"
        if (n % 5 === 0) outputString += "Plang"
        if (n % 7 === 0) outputString += "Plong"
        if (outputString === "") return n.toString() else return outputString
    }
}

File: lua/raindrops/raindrops.lua
return function(n)
	local str = ""

	if n % 3 == 0  then; str = str.."Pling" end;
	if n % 5 == 0  then; str = str.."Plang" end;
	if n % 7 == 0  then; str = str.."Plong" end;

	if str == "" then; return tostring(n); end
	return str
end
